### DISCUSSION TITLE
posts/2022-10-10-reactstate

### DISCUSSION THREAD

**Discussion by giscus** [cmdcolin/cmdcolin.github.io#8] [General]
# posts/2022-10-10-reactstate

Blogging for the future

https://cmdcolin.github.io/posts/2022-10-10-reactstate

**Comment by kmfb** [comment]
Life is shore, I use the [react-query](https://github.com/tanstack/query).

**Reply by kmfb** [reply]
short

**Reply by cmdcolin** [reply]
sure is! I added a react-query example to the footnotes, and an swr one too :)
